core/src/main/java/tech/pegasys/web3signer/core/service/jsonrpc/FcJsonRpc.java

```diff
@@ -33,22 +34,66 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.tuweni.bytes.Bytes;
+import org.hyperledger.besu.plugin.services.MetricsSystem;
+import org.hyperledger.besu.plugin.services.metrics.Counter;
+import org.hyperledger.besu.plugin.services.metrics.LabelledMetric;
+import org.hyperledger.besu.plugin.services.metrics.OperationTimer;
 
 @JsonRpcService
 public class FcJsonRpc {
+
   public static final int SECP_VALUE = 1;
   public static final int BLS_VALUE = 2;
 
   private static final Logger LOG = LogManager.getLogger();
 
   private final ArtifactSignerProvider fcSigners;
-
-  public FcJsonRpc(final ArtifactSignerProvider fcSigners) {
+  private final Counter secpSigningRequestCounter;
```

Comment:
I wonder if its a good idea to wrap all these Counter in FCCounters class to cleanup this class/constructor?

Reply:
then all we have to do is to inject one FCCounters instance instead.